File: obxod.py
import webbrowser
import os

# URL сайта, который нужно открыть
url = "https://www.youtube.com"

# Функция для открытия сайта в браузере по умолчанию
def open_site():
    webbrowser.open(url, new=2)

# Запуск функции
if __name__ == "__main__":
    open_site()

File: obxod.js
const express = require('express');
const request = require('request');
const app = express();
const port = process.env.PORT || 3000;

app.use((req, res) => {
    const url = `https://www.youtube.com${req.url}`;
    req.pipe(request(url)).pipe(res);
});

app.listen(port, () => {
    console.log(`Server is running on port ${port}`);
});
